### PR TITLE
Fix for the three space indent on using-shared-components branch.

### PR DESCRIPTION
**Pull request by andersonleite**
Fixes:

- Fix for the three space indent on using-shared-components branch.
- Update dependencies so typescript integration works

Fixes based on [this](https://twitter.com/garybernhardt/status/1091783301323345920?s=21) feedback.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "next": "^6.1.1",
-    "react": "^16.4.2",
-    "react-dom": "^16.4.2"
+    "next": "^7.0.2",
```
Comment (lucleray) [inline package.json]: We can update next to 8.0.0 now that it's out 🙂

File: package.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
...
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0"
```
Comment (lucleray) [inline package.json]: We can update this to 16.8 so we can use hooks if we want

**Review by lucleray — CHANGES_REQUESTED** [review]

**Comment by lucleray** [comment]
Additionally to this PR, I think we should, **for each branch of the repo** :
- use prettier (.prettierrc)
- update dependencies

**Comment by andersonleite** [comment]
@lucleray I updated all the other branches for the newest next and react branches.
About prettierrc, if it's ok for you, I will look at it later.

**Pull request closed without merging**